modules/nw-egressnetworkpolicy-object.adoc

```diff
@@ -49,7 +49,7 @@ endif::ovn[]
 [id="egressnetworkpolicy-rules_{context}"]
 == {kind} rules
 
-The following YAML describes an egress firewall rule object. The `egress` stanza expects an array of one or more objects.
+The following YAML describes an egress firewall rule object. The user can select either an IP address range in CIDR format, or a domain name, or use the `nodeSelector` to allow or deny egress traffic. The `egress` stanza expects an array of one or more objects.
```

Comment:
Is this two or three options? 
**either:** an IP address range in CIDR format, or a domain name
**_if not:_**  the `nodeSelector`  

Assuming  this, remove the first comma: IP address range in CIDR format or a domain name, or use the `nodeSelector`

Reply:
3 options so no or. Not sure why I worded that so awkwardly.